metabrainz/__init__.py

```diff
@@ -56,23 +55,23 @@ def create_app(config_path=None):
 
         app.config.from_pyfile(consul_config, silent=True)
 
+    # Printing out some debug values such as config and git commit
+    try:
+        with open(".git-version") as f:
```

Comment:
We could do this in `utils.get_git_commit`.

Reply:
Remind me, was this added for just this case or was this an existing function that works in other contexts?

Reply:
No, just added here, so could be removed easily.

Reply:
Done!